subject.py

```diff
@@ -0,0 +1,178 @@
+import datajoint as dj
+
+import reference
+
+schema = dj.schema(dj.config['names.%s' % __name__], locals())
+
+
+
+@schema
+class Species(dj.Manual):
+    # <class 'subjects.models.Species'>
+    definition = """
+    species_id:			int             # species id
+    ---
+    binomial:			varchar(255)	# binomial
+    display_name:		varchar(255)	# display name
+    """
+
+
+@schema
+class Strain(dj.Manual):
+    # <class 'subjects.models.Strain'>
+    definition = """
+    strain_id:			int             # strain id
+    ---
+    descriptive_name:		varchar(255)	# descriptive name
+    description:    		varchar(255)	# description
+    """
+
+
+@schema
+class Sequence(dj.Manual):
+    # <class 'subjects.models.Sequence'>
+    definition = """
+    sequence_id:		int             # sequence id
```

Comment:
technically this should be identified by the `base_pairs` but that's understandably inconvenient - if they have a better short name alternative (e.g. if their `informal_name` can be unique) then I'd suggest that be the primary key. Otherwise, it'll be fine to leave this identified by `sequence_id`.

Reply:
will xref to see if the informal name was infact unique upstream and adjust accordingly.

Reply:
ah in fact 'informal_name' was being used but via a django mechanism I didn't initially recognize as being a pkey. adjusted.